modules/julia/README.md

```diff
@@ -0,0 +1,84 @@
+OpenCV Julia Bindings
+============================
+This module contains some limited functionality that allows OpenCV functions be used from Julia. Upon installation the binding files are automatically registered with Julia's package manager like any normal package.
+
+This module requires Julia 1.4 and the CxxWrap.jl 0.10.
+
+CxxWrap Installation
+----
+Installation of CxxWrap is like any other Julia Package. Just start the REPL. Hit `]` and then type `add CxxWrap`.
+
+```
+$ julia
+...
+julia> ]
+pkg> add CxxWrap
+```
+
+For now, Julia module is only compatible with Ubuntu and MacOS. Also, you must use a source build of [libcxxwrap-julia](https://github.com/JuliaInterop/libcxxwrap-julia). Follow the link for instructions on how to do that.
+
+Build
+-----
+The Julia module is fully integrated into the OpenCV build system. While compiling add this to your command line `-DWITH_JULIA=ON`. If cmake finds a Julia executable available on the host system while configuring OpenCV, it will attempt to generate Julia wrappers for all OpenCV modules. If cmake is having trouble finding your Julia installation, you can explicitly point it to the Julia executable by defining the `Julia_EXECUTABLE` variable. For example:
+
+    cmake -DWITH_JULIA=ON -DJulia_EXECUTABLE=/home/user/julia-1.4.1/bin ..
+
+If you prefer using the gui version of cmake (cmake-gui), you can use the *Add Entry* option in the GUI to manually add the *path* variable `Julia_EXECUTABLE`.
+
+
+Install
+-------
+By default the Julia package is installed in `CMAKE_BINARY_DIR`, you can change this by setting the `JULIA_PKG_INSTALL_PATH` cmake variable. The package is automatically registered with the Julia package manager.
+
+Run
+---
+
+In order to use the bindings, simply type
+```bash
+$ julia
+...
+julia> using OpenCV
+```
+
+Note that this works only if you called `make install`. To run the wrapper package without making the installation target you must first set the environment variable `JULIA_LOAD_PATH` to the directory containing the OpenCV package. For example if in the build directory
+```bash
+$ export JULIA_LOAD_PATH=$PWD\OpenCV
```

Comment:
```suggestion
$ export JULIA_LOAD_PATH=$PWD/OpenCV
```

Reply:
Will do this change along with any other required revisions to not clutter commit history.

Reply:
@archit120, by the way:
1. [i]Note that this works only if you called `make install`[/i]
is it really true? I remember, I made it working without running `make install`

2. `export JULIA_LOAD_PATH=$PWD\OpenCV`
I think, even more correct would be to put
`export JULIA_LOAD_PATH=$PWD:$JULIA_LOAD_PATH`,
because OpenCV is the name of the package. It should not be a part of the load path

Reply:
1. You called `Pkg.develop` that's functionally equivalent to `make install`

2. The load path variable creates a virtual environment like with python. It needs to be pointed to the directory with the Project.toml file so this is correct aswell. In usual circumstances, this variable should not be set at all.

Reply:
Done